templates/license-checksum

```diff
@@ -0,0 +1,2 @@
+7a8086e36c141979221311787c1187f1091018b29239a3b168d90b36bc25f993  LICENSE
```

Comment:
Please remove this file, it's not needed. Having this file here will copy it to version 6 dir

Reply:
Done